content/11-serverless/02-serverless-comparison.mdx

```diff
@@ -306,6 +306,11 @@ Like Microsoft, Google seems to still be following Amazon's lead in the serverle
 
 Cloudflare Workers are capable of delivering high performance at a lower cost to developers due to their implementation details and some self-imposed limitations.  If your applications fit the Cloudflare Workers paradigm, it is a great choice for delivering low latency responses to your clients.
 
+<PrismaOutlinks>
```

Comment:
I think you might need another blank link beneath this one.  We had some rendering issues without that in the past.